### DISCUSSION TITLE
Map rotation option so that the map aligns to the device´s compass

### DISCUSSION THREAD

**Discussion by StefanLiebig** [organicmaps/organicmaps#6972] [Ideas]
This is a idea for a map rotation/orientation option that I only found in OsmAnd.
When hiking/walking I find it extremly helpfull when the map rotates when I turn
my smartphone so that it shows the "area" in direction the smartphone points to.

**Comment by patepelo** [comment]
Have you tried clicking in current location button for setting up? A line should appear locking the pointing of the triangle. This features is already a long time ago implemented. Maybe the question is if it isn't easy to discover.

**Reply by StefanLiebig** [reply]
Yes, that works. Great!
I probably expected it to be like in Osmand where there is a dedicated compass element/button with wich you can do that.
